src/components/search/site-search-form.tsx

```diff
@@ -10,15 +10,15 @@ const SiteSearchForm = ({...props}: ComponentProps<any>) => {
           Search this site
         </label>
         <input
-          className="rounded-full h-20 w-full lg:w-100 text-20"
+          className="rounded-full h-15 w-full lg:w-100 text-15 lg:border-black-20"
           type="text"
           placeholder="Search this site"
           id={inputId}
           name="q"
           required
         />
-        <button type="submit" className="absolute top-5 right-5">
-          <MagnifyingGlassIcon width={30}/>
+        <button type="submit" className="absolute top-2 right-5">
+          <MagnifyingGlassIcon width={30} color="#b1040e"/>
```

Comment:
Why aren't you using tailwind classes here?

Reply:
I did it because of this: https://stackoverflow.com/questions/69963915/react-heroicons-icons-are-always-black . Now I'm checking the documentation about using a color class.  Thanks for bringing my attention to it.

Reply:
`text-cardinal-red` or `fill-cardinal-red` should work. 
See this example: https://github.com/SU-SWS/cardinalsites-nextjs/pull/6/files#diff-cc03e1bb0d7321c2079f9354d527680f8c0d6e8db4e1c8e7e2ced00ee631ec20R25

Reply:
@pookmish I was thinking about it the wrong way.  I needed the `text-` in the class name.